pelican/server.py

```diff
@@ -91,7 +91,9 @@ def get_path_that_exists(self, original_path):
     def guess_type(self, path):
         """Guess at the mime type for the specified file.
         """
-        mimetype = server.SimpleHTTPRequestHandler.guess_type(self, path)
+        # strip trailing slash
+        path = path.rstrip().strip("/").strip(os.sep)
```

Comment:
Don't think `strip('/')` is wise here. Unix paths start with `/`. `rstrip` is probably safe though.

Reply:
Great catch. This has been swapped to `rstrip`.